src/pages/_app.js

```diff
@@ -1,5 +1,6 @@
-import '@/styles/globals.css'
+/* eslint-disable react/prop-types */
+import '../styles/globals.css';
```

Comment:
lets see if `/styles/global.css` work in place of this

Reply:
No, it's giving error. We'll set up absolute imports later.